workers/unity/Packages/com.improbable.gdk.core/Tests/Editmode/GameObjectRepresentation/SpatialOSBehaviourLibraryTests.cs

```diff
@@ -13,22 +15,9 @@ namespace Improbable.Gdk.Core.EditmodeTests
     [TestFixture]
     public class SpatialOSBehaviourLibraryTests
     {
-        // The dummy code needs to be removed once we have codegenerated writers.
         private const uint PositionComponentId = 54;
-        private const uint ComponentId2 = 1338;
-        private const uint ComponentId3 = 1339;
-
-        [WriterInterface]
-        [ComponentId(ComponentId2)]
-        private interface DummyWriter1
-        {
-        }
-
-        [WriterInterface]
-        [ComponentId(ComponentId3)]
-        private interface DummyWriter2
-        {
-        }
+        private const uint BlittableComponentId = 1001;
```

Comment:
May be a good reason to internally expose the codegenerate component ID to be use here

Reply:
No very nice way to do that at the moment though. (I think that ticket was/will be implemented on one of the other experimental branches?) What we could do is reflectively get the component ID attribute from the Reader or Writer interface. Should I do that?

Reply:
🤔 will be good to check existing tickets in JIRA for it or make one to be considered in grooming, for reflectively getting through readers: not worth it IMO

Reply:
Hmm this is a ticket in our sprint, but I'm pretty sure I saw it being implemented somewhere else https://improbableio.atlassian.net/browse/UTY-504 I'll ask around

Reply:
Correct, it's on the C# core branch https://github.com/spatialos/UnityGDK/blob/experimental/core-csharp-sdk/workers/unity/Packages/com.improbable.gdk.core/Components/ISpatialComponentData.cs . I guess we could make a one pointer ticket blocked on that merge to update this test with those IDs but it shouldn't be a serious issue anyway...

Reply:
Yes, but since that's already implemented in the C# Core experimental branch, I'd rather not create a duplicate implementation here. We made a ticket for changing this hard coded value once that's merged in: https://improbableio.atlassian.net/browse/UTY-857